larray_editor/editor.py

```diff
@@ -289,18 +281,10 @@ def setup_menu_bar(self):
         #===============#
         file_menu.addSeparator()
```

Comment:
this should be within the if qtconsole_available condition (otherwise you could get a double separator)